templates/base/storybook/.storybook/addons.js

```diff
@@ -0,0 +1,10 @@
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-viewport/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-docs/register';
+import '@storybook/addon-storysource/register';
+import '@storybook/addon-a11y/register';
+import 'storybook-addon-designs/register';
+import '@storybook/addon-jest/register';
```

Comment:
Lets drop config for following 
import '@storybook/addon-jest/register';
import '@storybook/addon-storysource/register';